static/src/js/util/map/__tests__/normalizeSpatial.test.ts

```diff
@@ -1,45 +1,6 @@
-import normalizeSpatial, {
-  getPolygonArea,
-  squareMetersToSquareKilometers
-} from '../normalizeSpatial'
+import normalizeSpatial from '../normalizeSpatial'
 
 describe('normalizeSpatial', () => {
-  describe('get polygon area', () => {
```

Comment:
did we move this or maybe I just forgot to remove these at some point?

Reply:
I removed it in favor of doing the calculation outside this util